apps/molgenis-components/src/components/forms/formUtils/formUtils.ts

```diff
@@ -206,31 +203,6 @@ export function executeExpression(
   return func(simplePostClient, ...Object.values(copy));
 }
 
-function isRefLinkWithoutOverlap(column: IColumn, values: Record<string, any>) {
-  if (!column.refLinkId) {
```

Comment:
Don't forget to also remove the unittest

Reply:
thx